powersimdata/data_access/tests/test_scenario_csv.py

```diff
@@ -104,30 +103,26 @@ def mock_row(sid=1):
     )
```

Comment:
Are we missing `runtime` and `infeasibilities` here?

Reply:
Yes, but also most of the columns could be omitted since they are not utilized by the storage logic.